content/types/_client_options.textile

```diff
@@ -14,28 +14,29 @@ h4.
 
-- <span lang="default">authUrl</span><span lang="ruby">:auth_url</span> := A URL that the library may use to obtain a token string (in plain text format), or a signed "@TokenRequest@":/realtime/types#token-request or "@TokenDetails@":/realtime/types#token-details (in JSON format). For example, this can be used by a client to obtain signed token requests from an application server.<br>__Type: @String@__
+- <span lang="default">authUrl</span><span lang="python">auth_url</span><span lang="ruby">:auth_url</span> := A URL that the library may use to obtain a token string (in plain text format), or a signed "@TokenRequest@":/realtime/types#token-request or "@TokenDetails@":/realtime/types#token-details (in JSON format). For example, this can be used by a client to obtain signed token requests from an application server.<br>__Type: @String@__
 
-- <span lang="default">authMethod</span><span lang="ruby">:auth_method</span> := _<span lang="default">@GET@</span><span lang="ruby">@:get@</span>_ The HTTP verb to use for the request, either <span lang="default">@GET@</span><span lang="ruby">@:get@</span> or <span lang="default">@POST@</span><span lang="ruby">@:post@</span><br>__Type: <span lang="default">@String@</span><span lang="ruby">@Symbol@</span>__
+- <span lang="default">authMethod</span><span lang="python">auth_method</span><span lang="ruby">:auth_method</span> := _<span lang="default">@GET@</span><span lang="ruby">@:get@</span>_ The HTTP verb to use for the request, either <span lang="default">@GET@</span><span lang="ruby">@:get@</span> or <span lang="default">@POST@</span><span lang="ruby">@:post@</span><br>__Type: <span lang="default">@String@</span><span lang="ruby">@Symbol@</span>__
 
-- <span lang="default">authHeaders</span><span lang="ruby">:auth_headers</span> := A set of key value pair headers to be added to any request made to the @authUrl@. Useful when an application requires these to be added to validate the request or implement the response.<br>__Type: <span lang="default">@Object@</span><span lang="ruby">@Hash@</span><span lang="php">@Associative Array@</span><span lang="java">@Param []@</span>__
+- <span lang="default">authHeaders</span><span lang="python">auth_headers</span><span lang="ruby">:auth_headers</span> := A set of key value pair headers to be added to any request made to the @authUrl@. Useful when an application requires these to be added to validate the request or implement the response.<br>__Type: <span lang="default">@Object@</span><span lang="ruby">@Hash@</span><span lang="php">@Associative Array@</span><span lang="java">@Param []@</span>__
```

Comment:
Think you're missing the type here, I assume it's not an `Object` for Python. Similar examples below